sdc/tests/tests_perf/generator.py

```diff
@@ -0,0 +1,104 @@
+import time
+from sdc.io.csv_ext import to_varname
+from typing import NamedTuple
+
+
+class TestCase(NamedTuple):
+    name: str
+    params: str
+    size: int = None
+    call_expression: str = None
+
+
+def gen(cases, method, class_add, type, prefix=''):
+    for params in cases:
+        test_params = TestCase(*params)
+        name = test_params.name
+        if test_params.params:
+            name += "_" + to_varname(test_params.params)
+        func_name = 'test_{}_{}_{}'.format(type, prefix, name)
+        func_name = to_varname(func_name).replace('__', '_')
```

Comment:
Can `func_name` consist of unavailable symbols or start with digit? Could you provide an example? If not this line is excess. Can you provide example where double underscore can be used?

Reply:
test_df_rolling_100__min

Reply:
When can you get such value? if `name == '_min'`?

Reply:
this name will be due to the parameters specified in the method so that the methods are not the same if we do several tests for one function.

Reply:
Could you provide an example of values `typ`, `prefix`, `name` when we can get double underscore?